src/client/v2/algod/compile.ts

```diff
@@ -28,7 +28,6 @@ export default class Compile extends JSONRequest<
     private source: string | Uint8Array
   ) {
     super(c);
-    this.source = source;
```

Comment:
why did it go? does TS initalize it implicitly?

Reply:
Yes, it is implicitly assigned. This is a typescript parameter property: https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties